docs-2.0/8.service-tuning/2.graph-modeling.md

```diff
@@ -68,6 +68,48 @@ nebula> MATCH (v1:player)-[e:temp]->() return collect(e.tmp);
 +----------------+
 ```
 
+### About dangling edges
+
+A dangling edge is an edge that only connects to a single vertex and only one part of the edge connects to the vertex.
+
+In Nebula Graph {{ nebula.release }}, dangling edges may appear in the following two cases.
+
+1. Insert edges with [INSERT EDGE](../3.ngql-guide/13.edge-statements/1.insert-edge.md) statement before the source vertex or the destination vertex exists.
+
+2. Delete vertices with [DELETE VERTEX](../3.ngql-guide/12.vertex-statements/4.delete-vertex.md) statement and the `WITH EDGE` option is not used. At this time, the system does not delete the related outgoing and incoming edges of the vertices. There will be dangling edges by default.
+
+Dangling edges may appear in Nebula Graph {{nebula.release}} as the design. And there is no MERGE statements of openCypher. The guarantee for dangling edges depends entirely on the application level. You can use [GO](../3.ngql-guide/7.general-query-statements/3.go.md) and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statements to find a dangling edge, but cannot use the [MATCH](../3.ngql-guide/7.general-query-statements/2.match.md) statement to find a dangling edge.
+
+Examples:
+
+```bash
+// Insert source vertex is '11', destination vertex is '13' and neither exists in the database
```

Comment:
Insert an edge that connects two vertices, with one vertex's ID being '11' and the other vertex's ID being '13', and the two vertices do not exist in the graph.